history/titles/000_wc_other_titles.txt

```diff
@@ -902,9 +910,12 @@ d_wardens = {
 	25.2.23={
 		holder=340021 #Maiev Shadowsong
 	}
-	604.10.12={
+	615.1.30={
 		holder=340101 #Sira Moonwarden
 	}
+	616.1.21={
+		liege=e_horde # Sira is raised as undead by Sylvanas
```

Comment:
I would rather say Sira stopped leading the Wardens and it's Maiev again.
https://wowpedia.fandom.com/wiki/The_Wardens

Reply:
Fixed in a73fedcf459b5b9f6d7920599cfffd5b7949cfe3.

Gave The Wardens titles (`d_wardens`, `c_lardonir`, and `c_sehyaeth`) back to Maiev after Sira is raised as undead.